lua/telescope/_extensions/gh_builtin.lua

```diff
@@ -111,6 +111,7 @@ B.gh_issues = function(opts)
         actions.select_default:replace(gh_a.gh_issue_insert)
         map("i", "<c-t>", gh_a.gh_web_view "issue")
         map("i", "<c-l>", gh_a.gh_issue_insert_markdown_link)
+        map("i", "<c-z>", gh_a.gh_develop "issue")
```

Comment:
Not sure what's the best key mapping here. `<C-b>` would be a bad choice for me because I use tmux and this is my tmux prefix.